PURL-TYPES.rst

```diff
@@ -582,6 +582,48 @@ swift
       pkg:swift/github.com/Alamofire/Alamofire@5.4.3
       pkg:swift/github.com/RxSwiftCommunity/RxFlow@2.12.4
 
+
+YOCTO Linux
+-----------
+``yocto`` for Yocto Linux recipes:
+
+There is no default package repository. The classifier `repository_url` is optional.
+
+:namespace: The namespace is the name of the layer which provides the recipe. The layer name as
+specified in the `BBFILE_COLLECTIONS` variable in `conf/layer.conf` of the layer.
+:name: The name of the package also known as [PN](https://docs.yoctoproject.org/ref-manual/variables.html#term-PN) in a yocto recipe.
```

Comment:
`BPN` might be more appropriate since it covers `target`, `native` and `nativesdk` builds